src/store/index.ts

```diff
@@ -299,7 +301,7 @@ export default (): Module<RootInterface, RootInterface> => ({
           // if any dataplanes are offline
           for (let i = 0; i < Object.values(result).length; i++) {
             const statusVal = Object.values(result[i])[0]
-            const isOnline = !(statusVal === 'Offline' || statusVal === 'offline')
+            const isOnline = !(statusVal === OFFLINE || statusVal === OFFLINE.toLowerCase())
```

Comment:
```suggestion
            const isOnline = statusVal.toLowerCase() !== OFFLINE.toLowerCase()
```
maybe like this?

Reply:
I have no idea if `statusVal` might be or not `undefined`. In the case of `undefined` it will fail so for now I will leave it as it was.

Reply:
got it